wg-charter.html

```diff
@@ -0,0 +1,364 @@
+        </p>
+        <p>
+          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>.
+        </p>
+        <p>
+          The group also welcomes non-Members to contribute technical submissions for consideration upon their agreement to the terms of the <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a>.
+        </p>
+        <p>
+          As stated above, the majority of the technical work for Working Group will take place in the <a href="https://www.w3.org/community/gpu/">GPU for the Web Community Group</a>.
+        </p>
+      </section>
+
+
+
+      <section id="communication">
+        <h2>
+          Communication
+        </h2>
+        <p id="public">
+          Technical discussions for this Working Group are conducted in <a href="https://www.w3.org/2018/Process-20180201/#confidentiality-levels">public</a>. Meeting minutes from teleconference and face-to-face meetings will be archived for public review, and technical discussions and issue tracking will be conducted in a manner that can be both read and written to by the general public. Working Drafts and Editor's Drafts of specifications will be developed on a public repository, and may permit direct public contribution requests.
+        </p>
+        <p>
+          Information about the group (including details about deliverables, issues, actions, status, participants, and meetings) will be available from the <a href="https://www.w3.org/gpu/">GPU for the Web Working Group home page.</a>
+        </p>
+        <p>
+          Most GPU for the Web Working Group teleconferences will focus on discussion of particular specifications, and will be conducted on an as-needed basis.
+        </p>
+        <p>
+          This group primarily conducts its technical work in its <a href="https://github.com/gpuweb">public repositories</a>. There is also a public mailing list <a id="public-name" href="mailto:public-gpu@w3.org">public-gpu@w3.org</a> (<a href="https://lists.w3.org/Archives/Public/public-gpu/">archive</a>).  The public is invited to contribute to the github repositories and post messages to the list. Regular activity summaries around the github repositories will be provided.
+        </p>
+      </section>
+
+
+
+      <section id="decisions">
+        <h2>
+          Decision Policy
+        </h2>
+        <p>
+          This group will seek to make decisions through consensus and due process, per the <a href="https://www.w3.org/2018/Process-20180201/#Consensus"> W3C Process Document (section 3.3</a>). Typically, an editor or other participant makes an initial proposal, which is then refined in discussion with members of the group and other reviewers, and consensus emerges with little formal voting being required.</p>
```

Comment:
I assume this is boilerplate that's also used for other groups but it is a bit weird to talk about proposals etc. given the WG would just stamp the specs.

Reply:
Yes, this is boilerplate.